system/include/libcxx/atomic

```diff
@@ -539,7 +539,8 @@ void atomic_signal_fence(memory_order m) noexcept;
 
 _LIBCPP_BEGIN_NAMESPACE_STD
 
-#if !__has_feature(cxx_atomic) && _GNUC_VER < 407
+// #if !__has_feature(cxx_atomic) && _GNUC_VER < 407 // XXX Emscripten: Added ' && !defined(__EMSCRIPTEN__)'
+#if !__has_feature(cxx_atomic) && _GNUC_VER < 407 && !defined(__EMSCRIPTEN__)
```

Comment:
Oh, thanks. I did test this, however I was tripped up by passing `--std=c++11`. Wrote up https://github.com/kripken/emscripten/issues/3938 about that. I'll correct the PR in a sec.